kube_downscaler/scaler.py

```diff
@@ -39,12 +39,14 @@ def autoscale_resource(resource: pykube.objects.NamespacedAPIObject,
     try:
```

Comment:
@hjacobs I'd like to make this work for namespaces as well, but it seems a bit clunky to add another param to the func signature (above this line). Perhaps a kwarg? (it might be clear if a few others above were also kwargs)

Reply:
It also means that when a namespace is excluded, instead of skipping the eval entirely, it has to re-eval everything in the namespace and scale them back up if necessary. Might be fine, but maybe there's a problem there I'm not thinking of